pkg/apis/softwarecomposition/scan_types.go

```diff
@@ -52,10 +52,10 @@ type ScannedControlStatus struct {
 type ScannedControlRule struct {
 	Name                  string
 	Status                RuleStatus
-	ControlConfigurations map[string][]string
+	ControlConfigurations map[string]string
```

Comment:
notable change

Reply:
@slashben this one is causing unmarshaling issues for `WorkloadConfigurationScan` (continuous scanning)
for some reason the protobufs don't like the initial map so I thought I could change it like that

Reply:
though it should be allowed https://protobuf.dev/programming-guides/proto3/#maps

Reply:
I think I've found a workaround by delaying unmarshalling with `json.RawMessage`